README.md

```diff
@@ -104,6 +104,16 @@ with non-English languages. `details.country` will still return country code.
 country = details.country # US
 country_name = details.country_name # United States
 ```
+##### EU Country
+
+`details.is_eu` will return true if the country is a member of EU , as supplied by the `eu.json` file.
+
+```ruby
+is_eu = details.is_eu # false
+```
+It is possible to change the file by setting the `eucountries` setting when creating the `IPinfo` object.
```

Comment:
Missing an extra newline for separation and let's please rename the setting itself:

```suggestion

It is possible to change the file by setting the `eu_countries` setting when creating the `IPinfo` object.
```